frontend/src/metabase/components/select-list/SelectListItem.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { ItemRoot, ItemIcon, ItemTitle } from "./SelectListItem.styled";
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  icon: PropTypes.string.isRequired,
+  onSelect: PropTypes.func.isRequired,
+  isHighlighted: PropTypes.bool,
+  hasRightArrow: PropTypes.bool,
+  variant: PropTypes.oneOf(["small", "medium"]),
+};
+
+const defaultProps = {
```

Comment:
`defaultProps` are deprecated in latest react, so let's inline these as default params

Reply:
Thanks, fixed that